src/reactA11yImgHasAltRule.ts

```diff
@@ -31,7 +38,8 @@ export class Rule extends Lint.Rules.AbstractRule {
   public static metadata: ExtendedMetadata = {
     ruleName: 'react-a11y-img-has-alt',
     type: 'maintainability',
-    description: 'Enforce that an `img` element contains the `alt` attribute or `role="presentation"` for decorative image.',
+    description: 'Enforce that an `img` element contains the non-empty `alt` attribute or' +
```

Comment:
This description is confusing, A or B and C is not easy to understand. Better be:
'Enforce that an `img` element contains the non-empty `alt` attribute. For decorative images, using empty alt attribute and `role="presentation"`.`